From: Martin Pitt <martin.pitt@ubuntu.com>
Date: Wed, 21 Jan 2015 15:26:13 +0100
Subject: util: Add some missing hidden_file() suffixes

dpkg itself also uses *.dpkg-dist, while .dpkg-{bak,backup,remove} are being
used by dpkg-maintscript-helper.
---
 src/shared/util.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/shared/util.c b/src/shared/util.c
index 26a4f72..7a79e3e 100644
--- a/src/shared/util.c
+++ b/src/shared/util.c
@@ -1518,6 +1518,10 @@ _pure_ static bool ignore_file_allow_backup(const char *filename) {
                 endswith(filename, ".dpkg-old") ||
                 endswith(filename, ".dpkg-new") ||
                 endswith(filename, ".dpkg-tmp") ||
+                endswith(filename, ".dpkg-dist") ||
+                endswith(filename, ".dpkg-bak") ||
+                endswith(filename, ".dpkg-backup") ||
+                endswith(filename, ".dpkg-remove") ||
                 endswith(filename, ".swp");
 }
 
